### PR TITLE
DX-1797: Track package by label id

### PR DESCRIPTION
**Pull request by rkrauskopf**
https://auctane.atlassian.net/browse/DX-1797

Unit tests will be completed as part of another JIRA task

### CODE REVIEW

File: src/track-by-label-id/format-response.ts [hunk 1]
```diff
@@ -1 +1,62 @@
...
+  response: Response.GetTrackingLogFromLabelResponseBody
+): TrackByLabelIdTypes.Response {
+  return formatTrackByLabelIdResult(response);
+}
+
+function formatTrackByLabelIdResult(
+  result: Response.GetTrackingLogFromLabelResponseBody
+): TrackByLabelIdTypes.TrackByLabelIdResult {
+  return {
+    trackingNumber: result.tracking_number || "",
```
Comment (j-r-j) [inline src/track-by-label-id/format-response.ts:19]: are we moving away from the pattern of making all results non optional?

**Review by j-r-j — COMMENTED** [review]
Just a question

File: src/track-by-label-id/format-response.ts (hunk 1 above)
Comment (rkrauskopf) [reply src/track-by-label-id/format-response.ts:19]: @j-r-j I had the same question on your `createLabel` PR. I'd like to sync up on this with you and Pierce after standup.

File: .eslintrc.yml [hunk 2]
```diff
@@ -7,6 +7,8 @@ ignorePatterns:
   - esm
   - cjs
   - node_modules
+  - "**/types/private-request.ts"
```
Comment (pierce-h) [inline .eslintrc.yml:10]: ❤️

File: src/track-by-label-id/validate-params.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * Performs client-side validation of the params passed in by the user.
+ */
+export function validateParams(params: TrackByLabelIdTypes.Params): void {
+  if (typeof params.labelId !== "string") {
+    throw new InvalidFieldValueError("Params", "must be a string.");
+  }
+
+  if (!params.labelId.startsWith("se-")) {
```
Comment (pierce-h) [inline src/track-by-label-id/validate-params.ts:12]: IMO I think we should avoid this type of validation for now

**Review by pierce-h — APPROVED** [review]

**Pull request merged**